modules/kubernetes-addons/aws-efs-csi-driver/locals.tf

```diff
@@ -41,9 +41,9 @@ locals {
   ]
 
   irsa_config = {
-    kubernetes_namespace              = local.namespace
+    kubernetes_namespace              = local.helm_config["namespace"]
```

Comment:
we should change line 4 to `try(var.helm_config.namespace, "kube-system")

and then leave this as `local.namespace`